### ISSUE TITLE
Package compiled aar in NPM

### ISSUE THREAD

**Issue by fourlastor** [react-native-community/react-native-svg#562]
As suggested in https://github.com/react-native-community/react-native-linear-gradient/issues/239.

TLDR; I think it would be great if instead of making the dev include the source code of the lib as a module in an android project, the lib would include the compiled artifact inside the NPM package.

This is already done by the react-native package (see http://facebook.github.io/react-native/docs/integration-with-existing-apps.html on Android) and offers multiple benefits, first of all it removes problems with different built tools versions between lib and app, avoids having to compile the code and in my experience worked much better with Android Studio (while the module approach would often fail the compilation because it wouldn't find the files in time).

**Comment by msand** [comment]
Closing until questions in https://github.com/react-native-community/react-native-linear-gradient/pull/255 are answered.
